src/Ethereum/Entry.cpp

```diff
@@ -7,22 +7,38 @@
 #include "Entry.h"
 
 #include "Address.h"
+#include "Ronin/Address.h"
 #include "Signer.h"
 
 using namespace TW::Ethereum;
 using namespace std;
 
 bool Entry::validateAddress(TWCoinType coin, const string& address, TW::byte, TW::byte, const char*) const {
-    return Address::isValid(address);
```

Comment:
I would prefer to keep ronin type addresses separate from ethereum to avoid any future bugs or confusion

Reply:
You meant add another entry point? Ronin is exactly ethereum with ronin: prefix, will still need to call ethereum code

Reply:
done

Reply:
Rather most thinkeble